samples/R10/test-production.sql

```diff
@@ -0,0 +1,32 @@
+---------- R10: Saboor ----------
+-- FILTER PLAYERS BY ONE OR MORE STATS --
+SELECT 
+    pid AS id, 
+    (firstName || ' ' || lastName) AS name,
+    games, points, assists, rebounds, season
+FROM Player 
+NATURAL JOIN PlayerStats 
+WHERE 
+    (rebounds >= 500 OR 500 IS NULL) AND
```

Comment:
What is the particular reason for these OR statements?

Reply:
When a user does not select any filtering for rebounds, rebounds = NULL.
Then NULL >= NULL is false & no entries show up. But we want all entries to show up.
We cannot use 0 because it excludes entries where REBOUNDS = NULL.
Consequently, need to include NULL IS NULL. 
Thought about using rebounds >= COALESCE(filterVar, rebounds) but it will not work when filterVar = NULL & rebounds = NULL. It would work if we can guarantee rebounds IS NOT NULL.